td-wx-download.xml

```diff
@@ -42,14 +42,20 @@
  <option value="IN9430">ACRE - Agronomy Center for Research and Education (ACRE) [Bowling]</option>
 <!-- <option value="NC0375">BATH_NEW - Bath Demonstration Site  [Youssef]</option>
  <option value="NC0375">BATH_A - Bath Demonstration Site  [Youssef]</option> -->
+ <option value="OH3915">AUGLA - Auglaize-E [Fausey]</option>
  <option value="IA0200">BEAR - Private Farm at Bear Creek [Jaynes]</option>
  <option value="MN0018">CLAY_C - Clay [Jia]</option>
  <option value="MN0018">CLAY_R - Clay [Jia]</option>
  <option value="MN0018">CLAY_U - Clay [Jia]</option>
+ <option value="OH1072">CRAWF - Crawford [Fausey]</option>
+ <option value="OH2098">DEFI_M - Defiance [Fausey]</option>
  <option value="OH2098">DEFI_R - Wetland Reservoir Subirrigation Systems (WRSIS) Demonst [Allred]</option>
  <option value="IN2825">DPAC - Davis Purdue Agricultural Center (DPAC) - West [Kladivko]</option>
  <option value="ND5754">FAIRM - Fairmount  [Jia]</option>
  <option value="OH8822">FULTON - Wetland Reservoir Subirrigation Systems (WRSIS) Demonst [Allred]</option>
+ <option value="OH4189">HARDIN - Hardin-N [Fausey]</option>
+ <option value="OH4189">HARDIN_NW - Hardin-NW [Fausey]</option>
+ <option value="OHC001">HENRY - Henry [Fausey]</option>
```

Comment:
Gio, sorry for this, but `OHC001` should not be used. It is a climate district average and not an actual point station.

Reply:
No problems, there are plenty of stations nearby